src/util/string.h

```diff
@@ -77,9 +77,9 @@ inline QString convertWCStringToQString(
     DEBUG_ASSERT(ilen >= 0); // unsigned -> signed
     switch (sizeof(wchar_t)) {
     case sizeof(ushort):
-        return QString::fromUtf16(reinterpret_cast<const ushort*>(wcs), ilen);
+        return QString::fromUtf16(reinterpret_cast<const char16_t*>(wcs), ilen);
     case sizeof(uint):
-        return QString::fromUcs4(reinterpret_cast<const uint*>(wcs), ilen);
+        return QString::fromUcs4(reinterpret_cast<const char32_t*>(wcs), ilen);
```

Comment:
It would also be nice to adjust the case targets.

Reply:
#4305